### PR TITLE
Update CSS token per new design system theming capability

### LINKED ISSUE #205: auro-dropdown: Theming Support
### Is your feature request related to a problem? Please describe.

This is regarding theming updates that need to happen to nearly all `auro` components

### Describe the solution you'd like

1. All tokens with the prefix `--auro` need to be renamed to `--ds` so they work with the new WCSS v5 update.
2. The WCSS package needs to be updated to ^5.0.0 and the import paths in the component *.scss files need to be updated as well

### Describe alternatives you've considered

None

### Additional context

Reference this `auro-button` pr https://github.com/AlaskaAirlines/auro-button/pull/231

### Exit criteria

All tokens referencing `--auro` have been updated.
Validate the changes after this work item is also complete

### PR DESCRIPTION
**Pull request by fajar-apri-alaska**
# Alaska Airlines Pull Request

**Fixes:** #205 

## Summary:

- Update the design tokens to reflect the new theming conventions regarding tokens from the AuroDesignTokens and JetstreamDesignTokens repo.

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/style.scss [hunk 1]
```diff
@@ -72,9 +89,9 @@
 
 .helpText {
   margin-top: 4px;
```
Comment (fajar-apri-alaska) [inline src/style.scss:85]: should I apply the tokens also for this hardcoded value?

File: package.json [hunk 2]
```diff
@@ -30,8 +30,9 @@
   },
   "devDependencies": {
     "@alaskaairux/design-tokens": "^3.13.0",
-    "@aurodesignsystem/webcorestylesheets": "^4.2.0",
+    "@aurodesignsystem/design-tokens": "^4.1.1",
```
Comment (blackfalcon) [inline package.json:34]: Remove the legacy `@alaskaairux/design-tokens` and also update the `peerDependencies` to reference the newer `@aurodesignsystem/design-tokens`

File: package.json [hunk 3]
```diff
@@ -30,8 +30,9 @@
...
     "@aurodesignsystem/eslint-config": "^1.3.0",
+    "@aurodesignsystem/webcorestylesheets": "^5.0.3",
```
Comment (blackfalcon) [inline package.json:36]: Update `peerDependencies` to reference the new `@aurodesignsystem/webcorestylesheets`

File: src/style.scss (hunk 1 above)
Comment (blackfalcon) [reply src/style.scss:85]: Not right now, we have a body of work coming to replace all remaining styles that have hard values.

File: src/style.scss [hunk 4]
```diff
@@ -1,3 +1,20 @@
+// Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license.
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// general WCSS imports
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/src/breakpoints';
```
Comment (blackfalcon) [inline src/style.scss]:
```suggestion

```

Resource reference not needed.

File: src/style.scss [hunk 5]
```diff
@@ -1,3 +1,20 @@
...
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/src/core';
+
+// Import Auro Sass variables
+// Dependency required to process variables from Icons
+// @import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
```
Comment (blackfalcon) [inline src/style.scss]:
```suggestion

```

References are not needed.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (fajar-apri-alaska) [reply package.json:34]: sure

File: package.json [hunk 6]
```diff
@@ -25,13 +25,13 @@
   },
   "peerDependencies": {
     "@alaskaairux/auro-icon": "^3.3.1",
-    "@alaskaairux/design-tokens": "^3.11.0",
-    "@aurodesignsystem/webcorestylesheets": "^4.2.0"
+    "@aurodesignsystem/design-tokens": "^4.1.1",
```
Comment (jason-capsule42) [inline package.json:29]: Both should be listed for now.
```suggestion
    "@alaskaairux/design-tokens": "^3.11.0",
    "@aurodesignsystem/design-tokens": "^4.1.1",
```

File: package.json (hunk 2 above)
Comment (jason-capsule42) [reply package.json:34]: After recent discussions on the Auro team we need to keep both for the time being per earlier comment.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

**Comment by fajar-apri-alaska** [comment]
I have pushed a new commit per your feedback @jason-capsule42. Thanks.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**